armi/reactor/blueprints/reactorBlueprint.py

```diff
@@ -191,8 +191,9 @@ def _constructComposites(self, cs, bp, loadComps, system, gridDesign):
             self._loadComposites(cs, bp, system, gridDesign.gridContents, gridDesign.orientationBOL)
 
             if isinstance(system, Core):
+                if not system.isFullCore:
+                    self._modifyGeometry(system, gridDesign)
```

Comment:
Are we sure that we want to skip all of `_modifyGeometry` when the system is full core, and not just the `scaleParamsRelatedToSymmetry` part?

Reply:
For the moment, "modify geometry" doesn't do anything else relevant. 

It could, perhaps, have a much less general-sounding name.

Reply:
Mike was right, this chunk of code is now being skipped, but it shouldn't be:
https://github.com/terrapower/armi/blob/38305de56bfa0d7d4ccdf9792560188ae6713eac/armi/reactor/blueprints/reactorBlueprint.py#L243-L251